src/cider/nrepl/middleware/util/java/parser.clj

```diff
@@ -216,10 +216,12 @@
   MethodDoc
   (parse-info [m]
     {:argtypes (mapv #(-> ^Parameter % .type typesym) (.parameters m))
-     :argnames (mapv #(-> ^Parameter % .name symbol) (.parameters m))})
+     :argnames (mapv #(-> ^Parameter % .name symbol) (.parameters m))
+     :type (str (. m returnType))})
```

Comment:
`(.returnType m)`

Reply:
I prefer `(. m returnType)`. In line with Java being a language designed for an IDE, you never type out:

```java
m.returnType
```

You type `m.` and get a list of completions based on the class of `m`.

I do the same in Clojure: type `(. m )` and get a list of completions for the class of `m`.

Reply:
But it's not like this in Clojure - as you don't really know the type of the receiver you get completions for possible classes anyways. At any rate - consistency in a codebase is important and we've always used the syntax suggested by @xiongtx so far, so please change this.